nservicebus/azure-service-bus/brokered-message-creation_header_abs_[,7).partial.md

```diff
@@ -0,0 +1,3 @@
+
+ 1. `NServiceBus.EnclosedMessageTypes`: Represents the message type expressed in a .Net native full typename specification (including namespace). E.g. `MyContracts.MyEvent`
+ 1. `NServiceBus.MessageIntent`: Represents what the message intent is, most likely the value has to be either `Send` or `Publish`.
```

Comment:
Would it add value to elaborate on the difference between the two? Publish would require a subscriber to exist, where Send would not.